10_integrations/dbt_modal_inference/duck_db_modal_inference.py

```diff
@@ -0,0 +1,276 @@
+# ---
+# cmd: ["modal", "run", "10_integrations/dbt_modal_inference/duck_db_modal_inference.py"]
+# ---
+# # dbt python model with llm inference
+#
+# In this example we demonstrate how you could combine dbt python models with llm
+# inference models powered by modal, allowing you to run serverless gpu workloads within dbt.
+#
+# ## Overview
+#
+# This example runs [dbt](https://docs.getdbt.com/docs/introduction) with a [duckdb](https://duckdb.org)
+# backend directly on top of modal, but could be translated to run on any dbt-compatible
+# database that supports python models. Similarly you could make these requests from UDFs
+# directly in SQL instead if you don't want to use python models.
+#
+# In this example we use an llm deployed in a previous example: [Serverless TensorRT-LLM (LLaMA 3 8B)](https://modal.com/docs/examples/trtllm_llama)
+# but you could easily swap this for whichever modal function you wish. We use this to classify the sentiment
```

Comment:
nit: we generally capitalize Modal and Function -- to separate the proper noun/product component "Function" from the normal old functions in Python